### DISCUSSION TITLE
Importing new midi mappings?

### DISCUSSION THREAD

**Discussion by Ksone1** [fayaaz/mixxx-pi-gen#79] [General]
Hello,

Just wanted to know how I would be able to import new midi mappings? I am on a raspberry pi 5 8gb so not sure if there will be an easier process for the latest update?

**Comment by fayaaz** [comment]
Import new mappings in the latest version of Mixxx or just use a new custom mapping?

If you want the latest mixxx - head to the actions and grab the[ latest .deb from the artifacts ](https://github.com/fayaaz/mixxx-pi-gen/actions/runs/8226364130)and install it with:
`sudo apt install ./<path_to_deb>`

If you want to use new midi mappings copy them to `/home/pi/.mixxx/controllers`

**Reply by jggraser** [reply]
how do I find the /home/pi area? there is nothing in the files when I look on my computer.

**Reply by fayaaz** [reply]
There's a few ways you can do this.

1. You can view the files from the terminal when you boot up an image. Win + Enter on your keyboard will create a new terminal. From here you could copy the files in the command line or open `firefox` and `nautilus` to copy the files by hand and GUI.
2. Copy the mappings onto a USB and plug this into the pi. Then use `nautilus` to copy them to /home/pi/.mixxx/controllers
3. Start SSH `systemctl --user start ssh` and use sftp to copy the files. Something like filezilla will be able to connect. The username is `pi` and password `mixxx`. You will need to connect the pi to a network for this. Wifi can be enabled using the `nmtui` command.

**Reply by jggraser** [reply]
How do I install nautilus

**Reply by fayaaz** [reply]
It should already be installed on the image. If not you need to run this in the terminal (which you can open with Mod + Enter - Mod is usually the windows key):

```
sudo apt install nautilus
nautilus
```

The second line runs the file manager.

**Reply by jggraser** [reply]
It keeps failing to install nautilus, do I need to enable wifi for it?

**Reply by fayaaz** [reply]
Yes you will need access to the internet somehow. Either wifi or ethernet. You can configure wifi using the `nmtui` command.

**Reply by jggraser** [reply]
After connecting to the internet it still says it has failed

**Reply by fayaaz** [reply]
`sudo apt update`
